### PR TITLE
added curly brackets to match the intructions

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)
The change is in line 113, the rest of the file is auto formatted.

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
- The way the function is being called is technically not correct...
- There's also a closely related ticket [here](https://www.notion.so/Consume-github-API-add-language-specific-examples-ea4acba515d2456f8a89a11c52b97af8?pvs=4) so maybe add calls for the other languages too?
- I think [the Person project instructions with example calls](http://syllabus.africacode.net/projects/oop/person/#:~:text=Example%20usage%3A,SSD%20hard%20drives.) are a good reference

**Comment by FaithMo** [comment]
> * The way the function is being called is technically not correct...
> * There's also a closely related ticket [here](https://www.notion.so/Consume-github-API-add-language-specific-examples-ea4acba515d2456f8a89a11c52b97af8?pvs=4) so maybe add calls for the other languages too?
> * I think [the Person project instructions with example calls](http://syllabus.africacode.net/projects/oop/person/#:~:text=Example%20usage%3A,SSD%20hard%20drives.) are a good reference

Ohhh of course, destructuring :) I'm dumb lol

Let me have a look at that ticket and discern what to do

File: content/projects/github-api-consume/part1/_index.md [hunk 1]
```diff
@@ -102,17 +102,51 @@ The function should return a list or array of pull requests on the repo such tha
...
+```bash
+# Python
+
+# input
+get_pull_requests({
+  owner="Umuzi-org",
+  repo="ACN-syllabus",
+  startDate="2022-03-01",
+  endDate=2022-03-10"
+})
```
Comment (Ngoakor12) [inline content/projects/github-api-consume/part1/_index.md]: python example should not have curly braces

File: content/projects/github-api-consume/part1/_index.md [hunk 2]
```diff
@@ -102,17 +102,51 @@ The function should return a list or array of pull requests on the repo such tha
...
+// Java
+
 # input
-get_pull_requests("Umuzi-org", "ACN-syllabus", "2022-03-01", "2022-03-10")
+getPullRequests(
+  owner,
+  repo,
+  startDate,
+  endDate
+)
```
Comment (Ngoakor12) [inline content/projects/github-api-consume/part1/_index.md]: this is a function call so you need to pass in actual values here... E.g:
```java
getPullRequests("Umuzi-org",....)
```

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

**Review by TheMaja — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**